.gitignore

```diff
@@ -23,4 +23,5 @@ packaging/rpm/shopify-cli.spec
 ext/shopify-extensions/shopify-extensions
 ext/javy/bin
 .console_history
-TODO.md
\ No newline at end of file
+TODO.md
+sorbet/rbi/hidden-definitions/errors.txt
```

Comment:
This line is not needed (given that we do not recommend using srb tooling and preferring Tapioca instead).